Makefile

```diff
@@ -71,13 +70,11 @@ $(MOSQUITTO_LIB): $(MOSQUITTO_DIR)
 
 # Build endpoint Legato app
 legato: cert
-	# Fetch the required external source code
-	# FIXME: Use 'fetch' instead of 'build' to avoid extra building actions.
-	#        The 'build' option is for getting the header file like 'mam/mam/mam_endpoint_t_set.h',
-	#        which can not be downloaded when the 'fetch' option is used.
-	bazel --output_base=$(OUTPUT_BASE_DIR) build //endpoint:libendpoint.so
-	# Generate endpoint Legato app
-	$(call platform-build-command)
+	# FIXME: Build the accelerator to get ta default logger from IOTA default
```

Comment:
Improve the descriptions regarding logger implementation. You have to address the fact of two existing logger implementations so far.